File: src/main/java/com/petmate/server/dto/OrganizationProfileDto.java
package com.petmate.server.dto;

import lombok.Data;
import java.util.List;

@Data
public class OrganizationProfileDto {
    private Long id;
    private Long userId;
    private String name;
    private String address;
    private String contact;
    private String description;
    private String logoUrl;
    private String status;

    private String orgType;
    private String verificationLevel;

    private Integer foundedYear;
    private String businessAddress;
    private String taxCode;
    private String establishmentNumber;
    private String website;
    private String fanpage;
    private String email;
    private String phone;

    private String representativeName;
    private String representativePhone;
    private String representativeEmail;
    private String representativeIdType;
    private String representativeIdNumber;
    private String representativeIdFrontUrl;
    private String representativeAvatarUrl;
    private String representativeSocialUrl;
    private String representativeRole;
    private String repLastVerifiedAt;

    private List<OrgDocumentDto> documents;

    private Boolean sterilizationPolicy;
    private Boolean vaccinationPolicy;
    private String policyDescription;
    private Boolean agreedTerms;

    private String adminNote;
    private String rejectionReason;
    private String verifiedAt;
    private String verifiedUntil;
    private Boolean isVerified;
    private String badgeLabel;

    private List<OrgMemberDto> members;
    private String ownerName;
}

File: app/src/main/java/com/example/petmate/model/OrganizationProfileDto.kt
package com.example.petmate.model

data class OrganizationProfileDto(
    val id: Long? = null,
    val userId: Long? = null,
    val name: String = "",
    val address: String = "",
    val contact: String = "",
    val description: String = "",
    val logoUrl: String? = null,
    val status: String = "PENDING",
    
    val orgType: String? = null,
    val verificationLevel: String? = null,
    
    val foundedYear: Int? = null,
    val businessAddress: String? = null,
    val taxCode: String? = null,
    val establishmentNumber: String? = null,
    val website: String? = null,
    val fanpage: String? = null,
    val email: String? = null,
    val phone: String? = null,
    
    val representativeName: String? = null,
    val representativePhone: String? = null,
    val representativeEmail: String? = null,
    val representativeIdType: String? = null,
    val representativeIdNumber: String? = null,
    val representativeIdFrontUrl: String? = null,
    val representativeAvatarUrl: String? = null,
    val representativeSocialUrl: String? = null,
    val representativeRole: String? = null,
    val repLastVerifiedAt: String? = null,
    
    val documents: List<OrgDocumentDto> = emptyList(),
    
    val sterilizationPolicy: Boolean? = null,
    val vaccinationPolicy: Boolean? = null,
    val policyDescription: String? = null,
    val agreedTerms: Boolean? = null,
    
    val adminNote: String? = null,
    val rejectionReason: String? = null,
    val verifiedAt: String? = null,
    val verifiedUntil: String? = null,
    val isVerified: Boolean = false,
    val badgeLabel: String? = null,
    
    val members: List<OrgMemberDto> = emptyList(),
    val ownerName: String? = null
)
